src/reanimated2/animation/util.ts

```diff
@@ -190,6 +202,105 @@ function decorateAnimation<T extends AnimationObject | StyleLayoutAnimation>(
     return finished;
   };
 
+  // START
+
+  const transformationMatrixOnStart = (
+    animation: Animation<AnimationObject>,
+    value: AffiniteMatrixFlat,
+    timestamp: Timestamp,
+    previousAnimation: Animation<AnimationObject>
+  ): void => {
+    const toValue = animation.toValue as AffiniteMatrixFlat;
```

Comment:
Do we really need `as AffiniteMatrixFlat` here?

Reply:
`Unfortunately yes, to assert that toValue has 16 elements`